### PR TITLE
Exasol (Non-Microsoft Publisher)

### PR DESCRIPTION
**Pull request by pj-spoelders**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Exasol/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+## Exasol - The analytics database connector
+
+Our superfast data analytics platform gives you the power, flexibility and scalability to meet the demands of your long-term data strategy. 
+
```
Comment (sriyen-msft) [inline certified-connectors/Exasol/README.md:5]: @pj-spoelders Please refer [the template for readme.md file](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and add required secitons.

File: certified-connectors/Exasol/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,524 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This connector enables you to work with the Exasol Database.",
```
Comment (sriyen-msft) [inline certified-connectors/Exasol/apiDefinition.swagger.json:4]: @pj-spoelders Please provide a more detailed description that should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

File: certified-connectors/Exasol/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+    "title": "Exasol",
+    "contact": {
+      "name": "Exasol",
+      "url": "https://github.com/exasol/exasol-rest-api/issues"
```
Comment (sriyen-msft) [inline certified-connectors/Exasol/apiDefinition.swagger.json:8]: @pj-spoelders Please provide contact email address

File: certified-connectors/Exasol/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/query/{query}": {
+      "get": {
+        "security": [
+          {
+            "ApiKeyAuth": []
```
Comment (sriyen-msft) [inline certified-connectors/Exasol/apiDefinition.swagger.json:42]: @pj-spoelders Adding a security key to each or one operation is not required and allowed. Can you please remove it.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Exasol/apiDefinition.swagger.json (hunk 4 above)
Comment (pj-spoelders) [reply certified-connectors/Exasol/apiDefinition.swagger.json]: I have removed these.

File: certified-connectors/Exasol/apiDefinition.swagger.json (hunk 3 above)
Comment (pj-spoelders) [reply certified-connectors/Exasol/apiDefinition.swagger.json]: Done.

File: certified-connectors/Exasol/apiDefinition.swagger.json (hunk 2 above)
Comment (pj-spoelders) [reply certified-connectors/Exasol/apiDefinition.swagger.json]: Hope this is OK.

File: certified-connectors/Exasol/README.md (hunk 1 above)
Comment (pj-spoelders) [reply certified-connectors/Exasol/README.md:6]: Thanks for bringing this template to our attention. I altered the document and added the missing sections.

**Comment by Amjed-Ayoub** [comment]
@pj-spoelders thank you for submitting the changes. @sriyen will review your request soon.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**